graylog2-plugin-interfaces/src/main/java/org/graylog2/plugin/buffers/MessageEvent.java

```diff
@@ -45,6 +46,7 @@ public MessageEvent newInstance()
 
     private RawMessage raw;
     private Message msg;
```

Comment:
That might make sense, but let's wait a little before making that change.
If message filters eventually want to synthesize messages on their own, they need a place to inject those, too.